### ISSUE TITLE
mex plugins should be responsible for their own grilo plugins

### ISSUE THREAD

**Issue by ghost** [media-explorer/media-explorer#179]
e.g.
The library plugin should load the grilo filesystem plugin
The tracker plugin should load the grilo tracker plugin
etc

**Comment by Cwiiis** [comment]
How will this work when two plugins want the same grilo plugin though? Perhaps plug-ins should be able to specify what grilo plugins they depend on and the application can load them via that?

**Comment by ghost** [comment]
Yeah, the mex plugin could check to see if the grilo plugin is already loaded or not by querying the GrlPluginRegistry

**Comment by Cwiiis** [comment]
Perhaps this should be a feature-request for Grilo? I can imagine it'd get
annoying if every time you want to load a Grilo plugin, you have to check
if it's already loaded first. Still like the dependency idea though, would
allow us to inform the user why a particular plug-in won't work (if the
issue is a missing or non-loading Grilo plug-in, at least)
